src/UIAutomation/Microsoft.UI.UIAutomation/Microsoft.UI.UIAutomation.idl

```diff
@@ -144,9 +144,10 @@ namespace Microsoft.UI.UIAutomation
 
     unsealed runtimeclass AutomationRemoteExtensionTarget : AutomationRemoteObject
     {
-        AutomationRemoteBool IsExtensionTarget();
+        /* Deprecated */ AutomationRemoteBool IsExtensionTarget();
```

Comment:
Does this need to be marked as deprecated? @mavangur could you also confirm

Reply:
I don't know when this got deprecated Rahul, At the time I introduced this API, we are in a view that it would be useful. I don't know who marked it deprecated.

Reply:
(Sorry for parachuting in, just hoping to provide some historic context :))

`Is<Type>` and `As<Type>` methods should be on `AutomationRemoteAnyObject`, since that's what facilitates "casting" between the generic "any" type and a "concrete" type. Basically it helps "inspect" the actual type and convert between them safely. (Sort of like a QI...)

This method was originally added to `AutomationRemoteExtensionTarget` where it doesn't do much good -- it's already representing a specific type. This PR is moving it to the right place and deprecating this one, presumably so that if anyone took a dependency on this so far for some reason they're not immediately broken.

Consider just introducing a (technically) breaking change and removing this, though. 

Mick also did have a question in the PR:

> Can someone please verify that the IsExtensionTarget op code in the remote ops VM really does ask if an anyObject is an extensionTarget (similar to all the other Is* op codes)?

I don't know for sure - I no longer have access to the Windows code base :( - but I would assume that it'd follow the same pattern as the others; I think someone would've had to go out of their way to implement it any differently given how things were originally structured.